### PR TITLE
Update __init__.py to better describe the app in the UI

### PR DESCRIPTION
**Pull request by jedelman8**
Updating to better inform user of what the app is/does...changes will be reflected in "Installed Plugins" in the UI.

image

### CODE REVIEW

File: nautobot_secrets_providers/__init__.py [hunk 1]
```diff
@@ -15,10 +15,10 @@ class NautobotSecretsProvidersConfig(PluginConfig):
     """Plugin configuration for the nautobot_secrets_providers plugin."""
 
     name = "nautobot_secrets_providers"
-    verbose_name = "Nautobot Secrets Providers"
+    verbose_name = "Secrets Management"
```
Comment (glennmatthews) [inline nautobot_secrets_providers/__init__.py:18]: Secrets management is a core Nautobot feature. Perhaps?

```suggestion
    verbose_name = "Secrets Management Extensions"
```

File: nautobot_secrets_providers/__init__.py [hunk 2]
```diff
@@ -15,10 +15,10 @@ class NautobotSecretsProvidersConfig(PluginConfig):
...
     version = __version__
     author = "Network to Code, LLC"
-    description = "Nautobot Secrets Providers Plugin."
+    description = "Nautobot App that provides direct integrations with Enterprise secrets management systems. Provides patterns to securely fetch secrets from other Nautobot Apps and Nautobot Jobs."
```
Comment (glennmatthews) [inline nautobot_secrets_providers/__init__.py:21]: Perhaps?

```suggestion
    description = "Integration with Enterprise secrets management systems. Provides patterns to securely fetch secrets for use by other Nautobot Apps and Nautobot Jobs."
```

Comment (jedelman8) [reply nautobot_secrets_providers/__init__.py:21]: Can we keep "Nautobot App..." as the lead-in for now?  It'll create some uniformity for now.
Good with "for use by" ...

Comment (jedelman8) [reply nautobot_secrets_providers/__init__.py]: I updated the second half of your proposed change.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**